processor/src/main/java/org/mapstruct/ap/internal/model/beanmapping/SourceReference.java

```diff
@@ -55,18 +56,30 @@
      */
     public static class BuilderFromMapping {
 
-        private Mapping mapping;
+       // private Mapping mapping;
```

Comment:
Why is this commented it out? If we don't need it, why not remove it?

Reply:
left over.. removed it.